Plan/common/src/main/resources/assets/plan/web/js/graphs.js

```diff
@@ -401,36 +406,40 @@ function formatTimeAmount(ms) {
     return out;
 }
 
-function sessionCalendar(id, events, firstDay) {
-    $(id).fullCalendar({
+function sessionCalendar(id, event_data, firstDay) {
+    window.calendars.sessions = new FullCalendar.Calendar(document.querySelector(id), {
+        themeSystem: 'bootstrap',
         eventColor: '#009688',
-        eventLimit: 4,
+        dayMaxEventRows: 4,
         firstDay: firstDay,
+        initialView: 'dayGridMonth',
 
-        eventRender: function (eventObj, $el) {
-            $el.popover({
-                content: eventObj.title,
+        eventDidMount: function (info) {
+            $(info.el).popover({
+                content: info.event.title,
                 trigger: 'hover',
                 placement: 'top',
                 container: 'body'
             });
         },
 
-        events: events,
+        events: function (fetchInfo, successCallback, failureCallback) {
+            successCallback(event_data)
+        },
 
         navLinks: true,
         height: 450,
         contentHeight: 445,
-        header: {
+        headerToolbar: {
```

Comment:
Is it possible to set a `bg-plan` class to color the buttons as the 'plan' color from https://github.com/plan-player-analytics/Plan/blob/master/Plan/common/src/main/resources/assets/plan/web/js/color-selector.js#L137 - The color selector might also require changes for night mode css related to the calendar ([`.fc-today`](https://github.com/plan-player-analytics/Plan/blob/master/Plan/common/src/main/resources/assets/plan/web/js/color-selector.js#L220))

Reply:
It looks like I can't add classes to the button but I can make the rule that applies the background color also include the header buttons.

Reply:
Sounds like a valid workaround